subgraphs/apeswap/schema.graphql

```diff
@@ -0,0 +1,557 @@
+  inputTokens: [Token!]!
+  " Output token of the pool. E.g. the UNI-LP token "
+  outputToken: Token!
+  " Amount of input tokens in the token's native unit "
+  inputTokenAmounts: [BigInt!]!
+
+  " Amount of output tokens in the token's native unit "
+  outputTokenAmount: BigInt!
+
+  " USD-normalized value of the transaction of the underlying (e.g. sum of tokens deposited into a pool) "
+  amountUSD: BigDecimal!
+
+  ##### DexAmm-Specific #####
+
+  " The pool involving this transaction "
+  pool: LiquidityPool!
+}
+
+type Withdraw implements Event @entity {
+  " { Transaction hash }-{ Log index }"
+  id: ID!
+
+  " Transaction hash of the transaction that emitted this event "
+  hash: String!
+
+  " Event log index. For transactions that don't emit event, create arbitrary index starting from 0 "
+  logIndex: Int!
+  " The protocol this transaction belongs to "
+  protocol: DexAmmProtocol!
+  " Address that received tokens "
+  to: String!
+  " Market that tokens are withdrawn from "
+  from: String!
+  blockNumber: BigInt!
+  timestamp: BigInt!
+  " Input tokens of the pool (not input tokens of the event/transaction). E.g. WETH and USDC from a WETH-USDC pool "
+  inputTokens: [Token!]!
+
+  " Output token of the pool (not output token of the event/transaction). E.g. the UNI-LP token "
+  outputToken: Token!
+
+  " Amount of input tokens in the token's native unit "
+  inputTokenAmounts: [BigInt!]!
+
+  " Amount of output tokens in the token's native unit "
+  outputTokenAmount: BigInt!
+
+  " USD-normalized value of the transaction of the underlying (e.g. sum of tokens withdrawn from a pool) "
+  amountUSD: BigDecimal!
+
+  ##### DexAmm-Specific #####
+
+  " The pool involving this transaction "
+  pool: LiquidityPool!
+}
+
+type Swap implements Event @entity {
+  " { Transaction hash }-{ Log index } "
+  id: ID!
+
+  " Transaction hash of the transaction that emitted this event "
+  hash: String!
+
+  " Event log index. For transactions that don't emit event, create arbitrary index starting from 0 "
+  logIndex: Int!
+  " The protocol this transaction belongs to "
+  protocol: DexAmmProtocol!
+  " Address that received tokens "
+  to: String!
+  " Market that tokens are withdrawn from "
+  from: String!
+  blockNumber: BigInt!
+  timestamp: BigInt!
+  " Token deposited into pool "
+  tokenIn: Token!
+  " Amount of token deposited into pool in native units "
+  amountIn: BigInt!
+  " Amount of token deposited into pool in USD "
+  amountInUSD: BigDecimal!
+  " Token withdrawn from pool "
+  tokenOut: Token!
+  " Amount of token withdrawn from pool in native units "
+  amountOut: BigInt!
+  " Amount of token withdrawn from pool in USD "
+  amountOutUSD: BigDecimal!
+  ##### DexAmm-Specific #####
+  " The pool involving this transaction "
+  pool: LiquidityPool!
+}
+
+############ Internal Use ############
+######################################
+# stores for USD calculations
+type Bundle @entity {
```

Comment:
prefix with underscore

Reply:
You should also rename Bundle to something more self-explanatory. See our Uniswap subgraph for reference

Reply:
fixed